crowbar_framework/app/controllers/installer/upgrades_controller.rb

```diff
@@ -263,9 +263,10 @@ def nodes_status
       respond_to do |format|
         format.json do
           render json: {
-            total: view_context.total_nodes_count,
+            total: view_context.all_non_admin_nodes_count,
```

Comment:
true that, I just looked at the `xyz_count` methods below and wanted to be consistent. but you're right it is not really needed